ipatests/test_integration/test_idviews.py

```diff
@@ -28,6 +28,9 @@ class TestCertsInIDOverrides(IntegrationTest):
     def uninstall(cls, mh):
         super(TestCertsInIDOverrides, cls).uninstall(mh)
         cls.master.run_command(['rm', '-rf', cls.reqdir], raiseonerr=False)
+        # Checking Traceback https://pagure.io/freeipa/issue/7958
+        http_logs = cls.master.get_file_contents(paths.VAR_LOG_HTTPD_ERROR)
```

Comment:
@varunmylaraiah 
This check is just applicable to TestCertsInIDOverrides test class and not applied to TestIDViews test class.
Better would be to move this to pytest_ipa/integration/tasks.py as utility function and use that in every test class's e.g in this case TestCertsInIDOverrides and TestIDViews.

Also test out these changes with help of temp_commit.yaml for these two test classes by including these two specific tests in temp_commit.yaml.